tests/integration/targets/legacy_missing_tests/aliases

```diff
@@ -0,0 +1,10 @@
+disabled
+
+# Lookup plugins
+aws_ssm  # covered by unit tests
+aws_service_ip_range
+aws_secret  # covered by unit tests
+aws_account_attribute
```

Comment:
https://github.com/ansible-collections/amazon.aws/tree/main/tests/integration/targets/lookup_aws_account_attribute

```suggestion
```

Reply:
Awesome! I was only looking for targets using only plugin's name 😅. Thank you.

Reply:
BTW, for all the other having lookup_name, I can verify that in a testing PR.

Reply:
No need for those even if they have called the the test suite like lookup_name.

Reply:
@tremble Can we merge this? Thanks.